src/main/java/seedu/address/model/person/GraduationDate.java

```diff
@@ -9,11 +9,12 @@
  */
 public class GraduationDate {
 
-    public static final String MESSAGE_CONSTRAINTS = "Graduation Date should be of the format month-year (MM-yyyy)"
+    public static final String MESSAGE_CONSTRAINTS = "Graduation Date should be of the format MONTH-YEAR (MM-yyyy)"
```

Comment:
It's a great idea to capitalize MONTH and YEAR as it makes it clearer for the user. Well done